data/schema.ttl

```diff
@@ -8606,7 +8606,7 @@ While such policies are most typically expressed in natural language, sometimes
     rdfs:label "skills" ;
     rdfs:comment "A statement of knowledge, skill, ability, task or any other assertion expressing a competency that is desired or required to fulfill this role or to work in this occupation." ;
```

Comment:
@philbarker rightfully mentions on #3381 that the description should be updated now the domain is extended.